<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="2" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="12559" started-at="2021-01-10T16:07:51Z" finished-at="2021-01-10T16:08:04Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="12559" started-at="2021-01-10T16:07:51Z" finished-at="2021-01-10T16:08:04Z">
      <class name="org.blazedemo.TestCode">
        <test-method status="PASS" signature="launch()[pri:0, instance:org.blazedemo.TestCode@cb5822]" name="launch" is-config="true" duration-ms="6795" started-at="2021-01-10T16:07:51Z" finished-at="2021-01-10T16:07:58Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- launch -->
        <test-method status="FAIL" signature="formfill(java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String, java.lang.String)[pri:0, instance:org.blazedemo.TestCode@cb5822]" name="formfill" duration-ms="0" started-at="2021-01-10T16:07:58Z" finished-at="2021-01-10T16:07:58Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Parameter 'inputName' is required by @Test on method formfill but has not been marked @Optional or defined
in C:\Users\Admin\AppData\Local\Temp\testng-eclipse--361176244\testng-customsuite.xml]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Parameter 'inputName' is required by @Test on method formfill but has not been marked @Optional or defined
in C:\Users\Admin\AppData\Local\Temp\testng-eclipse--361176244\testng-customsuite.xml
at org.testng.internal.Parameters.createParams(Parameters.java:290)
at org.testng.internal.Parameters.createParametersForMethod(Parameters.java:359)
at org.testng.internal.Parameters.createParameters(Parameters.java:620)
at org.testng.internal.Parameters.handleParameters(Parameters.java:769)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- formfill -->
        <test-method status="PASS" signature="test()[pri:0, instance:org.blazedemo.TestCode@cb5822]" name="test" duration-ms="5721" started-at="2021-01-10T16:07:58Z" finished-at="2021-01-10T16:08:04Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
      </class> <!-- org.blazedemo.TestCode -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
